### PR TITLE
[CCTRI-2639] - Fix minor divergences in mapping

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**
- Add private ip and ipv6 to targets
- Add internal field in sighting
- Add filter by private ip in search
- Fix flake issues

### CODE REVIEW

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Review by MaxMhonc — DISMISSED** [review]

File: code/api/observables.py [hunk 1]
```diff
@@ -52,12 +52,16 @@ def query(self, observable: str) -> dict:
...
         url = current_app.config['GUARD_DUTY_REFER_URL']
+        types = {
+            'ip': 'IP',
+            'ipv6': 'IPv6'
+        }
         return {
             'id': f'ref-aws-detective-search-ip-{value}',
-            'title': 'Search for this IP',
+            'title': f'Search for this {types[type_]}',
             'description': 'Check this IP with AWS Detective',
```
Comment (AnastasiiaRozlyvan) [inline code/api/observables.py:65]: Perhaps you need to remove `IP` from here too?

Comment (TarasMaliuchenko) [reply code/api/observables.py]: fixed - 901eb8a. Thank you

Comment (AnastasiiaRozlyvan) [reply code/api/observables.py]: Oh, and from `id`: ref-aws-detective-search-**ip**-{value}

Comment (TarasMaliuchenko) [reply code/api/observables.py]: Fixed - 38b6616

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Review by ytsek — APPROVED** [review]

**Pull request merged**